app/models/audit_log.rb

```diff
@@ -1,4 +1,6 @@
-class AuditLog < ApplicationRecord
+class AuditLog
```

Comment:
Why is this sticking around?

Reply:
Our logging functions for AuditLogs are all class methods on this model(`fetch_log, merge_log, and login_log`), and keeping this as a Model lets us continue to use the validations.

Reply:
Never mind I understand now.